wolfssl/wolfcrypt/settings.h

```diff
@@ -285,6 +285,11 @@
     #endif
 #endif
 
+#if defined(WOLFSSL_RENESAS_RA6M3G)
+    /* settings in user_settings.h */
+    #include <errno.h>
```

Comment:
What requires us to include errno.h here?

Reply:
errno.h is needed for wolfio.c, unless a #define for FREERTOS_TCP or WOLFSSL_RENESAS_RA6M3G is added to skip returning errno.  FreeRTOS has its own errors similiar to errno.h, but it does not seem to have a function that'll return the last error.

https://github.com/wolfSSL/wolfssl/blob/91a9117e1b7e7ca316299cef6dfc2243aa127651/src/wolfio.c#L91-L100

The errno define was moved to wolfio.h.

https://github.com/wolfSSL/wolfssl/blob/91a9117e1b7e7ca316299cef6dfc2243aa127651/wolfssl/wolfio.h#L152-L154

This could be changed to remove needing errno.h when building FREERTOS_TCP projects.